File: C/factorial.c
#include<stdio.h>
int main(){
    int i,n,f=1;
    printf("Enter the the number whose factorial is to be foud");
    scanf("%d",&n);
    for(i=1;i<=n;i++){
        f=f*i;
    }
    printf("The factorial of %d is %d",n,f);
    return 0;
}

File: Java/src/com/vaishakh/Basic/factorial.java
package com.vaishakh.Basic;

import java.util.Scanner;

public class factorial {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int f=1;
        System.out.println("Enter the number whose factorial is to be found");
        int n = sc.nextInt();
        for(int i = 1; i <= n; i++) {
            f=f*i;
        }
        System.out.println(f);
    }
}
